scalding-core/src/main/scala/com/twitter/scalding/CumulativeSum.scala

```diff
@@ -38,7 +38,13 @@ object CumulativeSum {
     /** Takes a sortable field and a monoid and returns the cumulative sum of that monoid **/
     def cumulativeSum(
       implicit sg: Semigroup[V], ordU: Ordering[U], ordK: Ordering[K]): SortedGrouped[K, (U, V)] = {
+      cumulativeSum(-1)
+    }
+
+    def cumulativeSum(reducers: Int)(
+      implicit sg: Semigroup[V], ordU: Ordering[U], ordK: Ordering[K]): SortedGrouped[K, (U, V)] = {
       pipe.group
+        .withReducers(reducers)
         .sortBy { case (u: U, _) => u }
         .scanLeft(Nil: List[(U, V)]) {
```

Comment:
why isn't this `Option[(U, V)]` again? I think this code would be less anxiety provoking then since the match would be clearer (I think it is due to flattenValues, if I recall).